modules/core/src/test/java/org/locationtech/jts/geom/GeometryImplTest.java

```diff
@@ -150,14 +150,18 @@ private void doTestEquals(Geometry a, Geometry b, boolean equalsGeometry,
     public void testInvalidateEnvelope() throws Exception {
         Geometry g = reader.read("POLYGON ((0 0, 0 50, 50 50, 50 0, 0 0))");
         assertEquals(new Envelope(0, 50, 0, 50), g.getEnvelopeInternal());
-        g.apply(new CoordinateFilter() {
-                public void filter(Coordinate coord) {
-                    coord.x += 1;
-                    coord.y += 1;
+        g.apply(new CoordinateSequenceFilter() {
```

Comment:
I think this change can be reverted as well.  The original code is is fine for testing the default implementation.  While it might be nice to test with a different CS representation, this should be done somewhere else.

Reply:
The [documentation to the CoordinateFilter](https://github.com/locationtech/jts/blob/master/modules/core/src/main/java/org/locationtech/jts/geom/CoordinateFilter.java#L25) discurages using it for modifying sequences.

Reply:
Good point!  Ok, let it stand...